src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1611,91 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+open ideal
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+-- a few lemmas to help shorten the proofs later
+lemma left_proj_quot_add_mk (x :R) (hx : x ∈ I) : ideal.quotient.mk (I ⊔ J) x = 0 :=
```

Comment:
You only use this once. I would simply inline the proof.

Reply:
I removed the lemma, and used `lift_mk_of_le` instead